.env.example

```
@@ -0,0 +1,158 @@
+BETTER_STACK_API_KEY=""
+
+# -----------------------------------------------------------------------------
+# VERCEL (AUTO-INJECTED IN VERCEL DEPLOYMENTS)
+# -----------------------------------------------------------------------------
+# These are automatically set by Vercel - no need to configure locally
+VERCEL_BRANCH_URL=""
+VERCEL_PROJECT_PRODUCTION_URL=""
+VERCEL_REGION=""
+VERCEL_DEPLOYMENT_ID=""
+VERCEL_PROJECT_ID=""
+VERCEL=""
```

dotenv-linter (4.0.0)

[warning] [EndingBlankLine] No blank line at the end of the file

(EndingBlankLine)